### PR TITLE
Update ItemCooldowns.java

### PR DESCRIPTION
**Pull request by VeritasDL**
add legendary monkey pet wood cutting cooldown attempt

idk if this is right or if im calling the api every time a block's broke lol

### CODE REVIEW

**Comment by VeritasDL** [comment]
updated to use the async stuff and to use a 3 min cooldown for api requests

File: src/main/java/de/hysky/skyblocker/skyblock/item/ItemCooldowns.java [hunk 1]
```diff
@@ -17,40 +24,109 @@
...
-                if (!isOnCooldown(JUNGLE_AXE_ID)) {
-                    ITEM_COOLDOWNS.put(JUNGLE_AXE_ID, new CooldownEntry(2000));
-                }
-            } else if (usedItemId.equals(TREECAPITATOR_ID)) {
-                if (!isOnCooldown(TREECAPITATOR_ID)) {
-                    ITEM_COOLDOWNS.put(TREECAPITATOR_ID, new CooldownEntry(2000));
+            currentCooldown();
+            if (usedItemId.equals(JUNGLE_AXE_ID) || usedItemId.equals(TREECAPITATOR_ID)) {
+                if (!isOnCooldown(JUNGLE_AXE_ID) || !isOnCooldown(TREECAPITATOR_ID)) {
+                    ITEM_COOLDOWNS.put(usedItemId, new CooldownEntry((int) currentCooldown));
```
Comment (viciscat) [inline src/main/java/de/hysky/skyblocker/skyblock/item/ItemCooldowns.java:121]: I think `currentCooldown()` should be moved in the ifs below. It's useless doing an API call if they aren't using a treecapitator

**Review by viciscat — DISMISSED** [review]

**Comment by kevinthegreat1** [comment]
Hi, this pr will require some more work. We have many pr's right now so it may be a bit when I get to this.

**Comment by kevinthegreat1** [comment]
Cleaned up the profile stuff a lot. Did not test yet.

**Review by viciscat — COMMENTED** [review]
Looks like this should work but i don't have legendary monke so I can't test

**Review by kevinthegreat1 — APPROVED** [review]
Tested the profile part.

**Comment by kevinthegreat1** [comment]
Hi, the commit history is a bit messed up. Can you run a `git rebase origin/master`? If you don't know how to rebase and you're fine if I commit the changes, then I can do it for you.

**Comment by VeritasDL** [comment]
> Hi, the commit history is a bit messed up. Can you run a `git rebase origin/master`? If you don't know how to rebase and you're fine if I commit the changes, then I can do it for you.

i think i did it? idk

**Comment by viciscat** [comment]
oof I don't think you did it right... You aren't supposed to merge after you rebase. You are supposed to force push `git push --force` if I remember correctly

**Review by kevinthegreat1 — APPROVED** [review]

**Pull request merged**